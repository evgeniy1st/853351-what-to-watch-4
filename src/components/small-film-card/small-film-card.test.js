import React from "react";
import renderer from "react-test-renderer";
import SmallFilmCard from "./small-film-card.jsx";

it(`render SmallFilmCard`, () => {
  const mockFilm =
    {
      title: `Fantastic Beasts: The Crimes of Grindelwald`,
      image: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`
    };

  const tree = renderer
    .create(<SmallFilmCard
      key={mockFilm.title}
      film={mockFilm}
      onTitleClick={() => {}}
      onCardHover={() => {}}
    />)
    .toJSON;

  expect(tree).toMatchSnapshot();
});
