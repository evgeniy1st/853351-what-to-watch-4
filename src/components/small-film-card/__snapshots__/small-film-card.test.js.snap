// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render SmallFilmCard 1`] = `[Function]`;
